### PR TITLE
Checkpoint 1

### PR DESCRIPTION
**Pull request by thehdjs**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -173,59 +173,59 @@ struct CarWash //2)
...
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)weight
+    2)size
+    3)price
+    4)saw
+    5)pliers
```
Comment (matkatmusic) [inline main.cpp]: how about `number of cutting tools`?
how about `number of grabbing tools`?
`saw` and `pliers` fall into the `has a ___` category of properties. The instructions say to not write those kinds of properties.
- [ ] revise.

File: main.cpp [hunk 2]
```diff
@@ -173,59 +173,59 @@ struct CarWash //2)
...
+    2)faders
+    3)inputs
+    4)effects
+    5)outputs
 3 things it can do:
-    1)
-    2)
-    3)
+    1)mix
+    2)embed
```
Comment (matkatmusic) [inline main.cpp]: I'm not sure what `embed` means in this context, so how about `fade to black`?
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (thehdjs) [reply main.cpp]: Indeed hadn't read that far

File: main.cpp (hunk 2 above)
Comment (thehdjs) [reply main.cpp]: its embedding sound to the picture

**Review by matkatmusic — APPROVED** [review]
Cool, onward to checkpoint 2

**Review by matkatmusic — APPROVED** [review]
Onward to checkpoint 3!

**Review by matkatmusic — COMMENTED** [review]
Ok, code it up!  Don't use arrays.

File: main.cpp [hunk 3]
```diff
@@ -186,6 +185,35 @@ struct CarWash //2)
...
+    unsigned int length = 150;
+    //
+    float price = 75;
+    
+    struct Acut
+    {
+    float cutDepth = 10;
+    float cutLength = 50;
+
+    void hurt(int pain);
```
Comment (matkatmusic) [inline main.cpp]: - [x] Fix the indentation for these member variables

File: main.cpp [hunk 4]
```diff
@@ -186,6 +185,35 @@ struct CarWash //2)
...
+    // smallest thickness in mm
+    float sharpness = 0.3f;
+    // biggest thickness
+    float thickness = 1.5f;
+    //length in mm
+    unsigned int length = 150;
+    //
+    float price = 75;
+    
+    struct Acut
```
Comment (matkatmusic) [inline main.cpp]: I think a better name would be `Incision`
- [x] revise the class name.

File: main.cpp [hunk 5]
```diff
@@ -298,6 +473,35 @@ struct CarWash //2)
...
+    char classType = 'A';
+    //caracter
+    int caracter = 0;
+    
+    struct ColorType
+    {
+    int age;
+    float depth;
+
+    void createHarmonics(int harmonicType);
```
Comment (matkatmusic) [inline main.cpp]: - [x] Fix the indentation

**Review by matkatmusic — CHANGES_REQUESTED** [review]
This is good.  Just a few minor changes requested

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**